### ISSUE TITLE
Current 'requests' release breaks httpie

### ISSUE THREAD

**Issue by mverteuil** [jakubroztocil/httpie#339]
Please change the setup.py to pin against the latest version of requests that is compatible instead of letting requests move up to 2.6.0 (or later) which breaks httpie.

I know that requests works @ 2.3.0, but later versions may be compatible too.

**Comment by mverteuil** [comment]
For reference, the traceback seen with requests 2.6.0 is:
```traceback
 ❯ http                                                                                                                                                                                    [3:52:36 pm]
Traceback (most recent call last):
  File "/usr/local/bin/http", line 7, in <module>
    from httpie.__main__ import main
  File "/usr/local/lib/python2.7/site-packages/httpie/__main__.py", line 6, in <module>
    from .core import main
  File "/usr/local/lib/python2.7/site-packages/httpie/core.py", line 21, in <module>
    from .compat import str, is_py3
  File "/usr/local/lib/python2.7/site-packages/httpie/compat.py", line 6, in <module>
    from requests.compat import (
ImportError: cannot import name is_windows
```

**Comment by jakubroztocil** [comment]
This problem should be fixed in the latest version of HTTPie (0.9.2).

**Comment by mverteuil** [comment]
Sweet, thanks :)
